src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -2055,9 +2055,14 @@ namespace WaterToAirHeatPumpSimple {
                 } else if (simpleWatertoAirHP.WatertoAirHPType == "COOLING") {
 
                     //       use companion heating coil capacity to calculate volumetric flow rate
-                    if (simpleWatertoAirHP.CompanionCoolingCoilNum > 0) {
-                        SystemCapacity =
-                            state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionCoolingCoilNum).RatedCapHeat;
+                    if (simpleWatertoAirHP.CompanionHeatingCoilNum > 0) {
+                        if (state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionHeatingCoilNum).RatedCapHeat ==
+                            DataSizing::AutoSize) {
+                            SystemCapacity = simpleWatertoAirHP.RatedCapCoolTotal;
+                        } else {
+                            SystemCapacity =
+                                state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionHeatingCoilNum).RatedCapHeat;
+                        }
```

Comment:
Fix the companion coil reference and handle cases where the heating companion coil has not yet been sized.